docker-java-api/src/main/java/com/github/dockerjava/api/command/EventsCmd.java

```diff
@@ -33,6 +34,12 @@
      */
     EventsCmd withEventFilter(String... event);
 
+    /**
+     * @param eventTypes
+     *            - event types to filter
+     */
+    EventsCmd withEventTypeFilter(EventType... eventTypes);
```

Comment:
Let's make this a default method and add `withEventTypeFilter(String... eventTypes)`, so that newly added event types will not require an update in docker-java